source/_components/climate.generic_thermostat.markdown

```diff
@@ -35,8 +35,9 @@ Configuration variables:
 - **ac_mode** (*Optional*): Set the switch specified in the *heater* option to be treated as a cooling device instead of a heating device.
 - **min_cycle_duration** (*Optional*): Set a minimum amount of time that the switch specified in the *heater* option must be in it's current state prior to being switched either off or on.
 - **tolerance** (*Optional*): Set a minimum amount of difference between the temperature read by the sensor specified in the *target_sensor* option and the target temperature that must change prior to being switched either off or on. For example, if the target temperature is 25 and the tolerance is 0.5 the heater will start when the sensor goes below 24.5 and it will stop when the sensor goes above 25.5.
+- **keep_alive** (*Optional*): Set a keep-alive interval. If set, the switch specified in the *heater* option will be  triggered every time the interval elapses. Use with heaters and A/C units that shut off if they don't receive a signal from their remote for a while.
```

Comment:
Oops.
I didn't notice this comment until after the pull request has been merged. I guess making a new pull request just for this is a bit over the top. But I'm planning on making more changes to the Generic Thermostat in the future - specifically, adding support for switching between cooling and heating in the UI, thus obsoleting the ```ac_mode``` property. I'll remove the extra space as part of the changes in the documentation required then.

Meanwhile, I assume that in the browser, it'll render with a single space anyway.